### DISCUSSION TITLE
Parametric search

### DISCUSSION THREAD

**Discussion by iodize6399** [inventree/InvenTree#4057] [Q&A] [answered]
Is there a way to filter parts using the already assigned parameter values? Something similar to what Digikey does, wherein it presents a filter table on top which lists all the associated options with that parameter.

**Comment by SchrodingersGat** [comment, marked as answer]
This is currently not implemented. Note that we have plans for significant improvements to how parametric filtering is done: https://github.com/inventree/InvenTree/issues/3080

However as there is a large list of outstanding tasks, this will not be tackled soon unless someone is keen to implement or sponsor the feature :)

**Reply by eeintech** [reply]
Oh I can't wait!
